test-client/test-server/getAccessToken.js

```diff
@@ -14,7 +14,7 @@ module.exports = () => {
   console.log('Getting Access Token for user invite.');
   return new Promise((resolve, reject) => {
     wreck.post('/op/token', {
-      payload: `grant_type=client_credentials&scope=admin`,
+      payload: `grant_type=client_credentials&scope=admin&scope=superadmin`,
```

Comment:
Probably don't need to test for the `superadmin` scope here, I don't know if the test-server is going to have a client with that permission available